CRM/Report/Form/Contribute/Bookkeeping.php

```diff
@@ -289,6 +289,7 @@ public function __construct() {
         'fields' => [
           'receive_date' => [
             'default' => TRUE,
+            'name' => 'contribution_receive_date',
```

Comment:
Note these changes are needed because https://github.com/civicrm/civicrm-core/blob/master/CRM/Report/Form.php#L760 uses the fields() function on the BAO rather than API getfields call